R/pkg/inst/profile/shell.R

```diff
@@ -47,5 +47,5 @@
   cat("    /_/", "\n")
   cat("\n")
 
-  cat("\n SparkSession available as 'spark'.\n")
+  cat("\nSparkSession available as 'spark'.\n")
```

Comment:
i thought this was here to align with the previous lines?

Reply:
Yea, it's aligned with previous lines. Didn't see empty spaces in other shell cmd(spark-shell, pyspark). I'll close this if it's confirmed not necessary.

Reply:
Can you show other shell messages and explain why this change makes it consistent 8n the PR description?

Reply:
Updated the description.